testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -1649,7 +1649,8 @@ enum Signal {
 #[test]
 #[ignore]
 fn atlas_integration_test() {
-    if env::var("BITCOIND_TEST") != Ok("1".into()) {
+    // to run this test, BITCOIND_TEST=1 and ATLAS_TEST=1 must be set
+    if env::var("BITCOIND_TEST") != Ok("1".into()) || env::var("ATLAS_TEST") != Ok("1".into()) {
```

Comment:
Ha, do you mind if we go with
https://github.com/blockstack/stacks-blockchain/pull/2173/files#diff-e782414dee532a3fcdf6a65f99f5d6455c8773d7511827ed3011082d0440976b
as a patch? I'm sleeping thanks to the test :)

Reply:
Sure -- will update.